src/item.cpp

```diff
@@ -1927,12 +1927,14 @@ std::string item::info( std::vector<iteminfo> &info, const iteminfo_query *parts
         }
         if( g->u.has_identified( typeId() ) ) {
             if( book.skill ) {
-                if( g->u.get_skill_level_object( book.skill ).can_train() &&
+                const SkillLevel &skill = g->u.get_skill_level_object( book.skill );
+                if( skill.can_train() &&
                     parts->test( iteminfo_parts::BOOK_SKILLRANGE_MAX ) ) {
```

Comment:
Just a tiny nitpick: there's no need for the line break anymore, this can fit in one line now.

Reply:
Oh, yeah. I missed that while I was going over it last time (I did make an effort to collapse lines and let astyle re-evaluate what gets split, but I wasn't too thorough)

I can grab that after I finish my current project.